explainability-core/src/main/java/org/kie/trustyai/explainability/local/shap/ShapResults.java

```diff
@@ -78,68 +78,72 @@ public int hashCode() {
         return Objects.hash(Arrays.hashCode(saliencies), fnull);
     }
 
+    /**
+     * Represent the ShapResult as a string
+     *
+     * @return ShapResult string
+     */
     @Override
     public String toString() {
         return toString(3);
     }
 
+    /**
+     * Represent the ShapResult as a string
+     * 
+     * @param decimalPlaces The decimal places to round all numeric values in the table to
+     *
+     * @return ShapResult string
+     */
     public String toString(int decimalPlaces) {
```

Comment:
Feel free to align this method name if you decide to rename the `toString()` method